internal/importers/base/importer_test.go

```diff
@@ -14,7 +15,17 @@ import (
 	"github.com/golang/dep/internal/test"
 )
 
+func parallelAllowed() bool {
+	// Disable parallel tests on Windows until we can resolve this error:
+	// "remote repository at https://github.com/carolynvs/deptest-importers does not exist, or is inaccessible"
+	return runtime.GOOS != "windows"
+}
+
 func TestBaseImporter_IsTag(t *testing.T) {
+	if parallelAllowed() {
```

Comment:
I don't know enough about subtests to understand if we need to call parallel in both the root test and the subtests, or if just the subtests are enough? Do you have more experience with this or a link so I can RTFM? 😊